db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Operations/StateMachines/ResizeRequest.class/__methods__/pending_request.rb

```diff
@@ -0,0 +1,11 @@
+# Second step - wait until instance get ready for resize confirm
+
+vm = $evm.root['vm']
+
+if vm.validate_resize_confirm
```

Comment:
@aufi 
Would this work if there are 2 concurrent resize requests, do we need to keep the notion of a task and track when the task ends and then check vm.validate_resize_confirm